src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,30 +1,50 @@
-import React, { PropTypes } from 'react';
+/* global FB, location */
+import React, { Component, PropTypes } from 'react';
 import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
 import FaTwitter from 'react-icons/lib/fa/twitter';
 import styles from './SocialBtns.sass';
+import fbSdk from './fb_sdk';
 
-const SocialBtns = ({
-  fbLink = 'https://www.facebook.com',
-  twitterLink = 'https://twitter.com',
-  iconSize = '25'
-}) => {
-  return (
-    <div className={styles.socialBtns}>
-      <a
-        href={fbLink}
-        className={styles.facebookBtn}
-      >
-        <FaFacebookSquare size={iconSize} />
-        <span>Share on Facebook</span
-      >
-      </a>
-      <a href={twitterLink} className={styles.twitterBtn}>
-        <FaTwitter size={iconSize} />
-        <span>Share on Twitter</span>
-      </a>
-    </div>
-  );
-};
+class SocialBtns extends Component {
+
+  componentDidMount() {
+    fbSdk.init();
+  }
+
+
+  render() {
+    const eventLink = location.href;
+    const twitterLink = `https://twitter.com/intent/tweet?text=${eventLink}`;
+    const iconSize = '25';
+
+    function fbShareHandler(event) {
+      event.preventDefault();
+      FB.ui({
+        method: 'share',
+        href: location.href,
+      });
+    }
+
+    return (
+      <div className={styles.socialBtns}>
+        <a
+          href={eventLink}
+          className={styles.facebookBtn}
+          onClick={fbShareHandler.bind(this)}
+        >
+          <FaFacebookSquare size={iconSize} />
+          <span>Share on Facebook</span
+        >
+        </a>
+        <a href={twitterLink} className={styles.twitterBtn}>
+          <FaTwitter size={iconSize} />
```

Comment:
I unfortunately don't have time to pull down and run your PR, but I was thinking it would be ideal if this link opened up a modal window for sharing, or at the very least, opened in a new browser tab, just to make sure that the user isn't taken away from our site.